packages/client/katello-host-tools/katello-host-tools.spec

```diff
@@ -170,8 +170,23 @@ rm -rf %{buildroot}
 %global plugins_confdir %{_sysconfdir}/yum/pluginconf.d
 %endif
 
+pushd src
+%if %{dnf_install} || 0%{?suse_version} >= 1500
+%py3_install
+%else
+%if 0%{?rhel} == 6
+%{__python} setup.py install
+%else
+%py_build
+%endif
+%endif
+popd
+
 mkdir -p %{buildroot}%{katello_libdir}
 mkdir -p %{buildroot}%{plugins_dir}
+mkdir -p %{buildroot}%{_sbindir}
+
+mv %{buildroot}%{_bindir}/* %{buildroot}%{_sbindir}/
```

Comment:
Looks like this is failing on EL7.

Reply:
It appears I didn't update line 184 in the first commit, so it was still calling %py_build instead of %py_install.  Should be fixed now.